static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -66,11 +66,14 @@ const findBreakpoint = (): string => {
 };
 
 const inskinTargetting = (): string => {
-    if (storage.local.get('gu.hasSeenPrivacyBanner')) {
-        const vp = getViewport();
-        if (vp && vp.width >= 1560) return 't';
+    const vp = getViewport();
+    if (vp && vp.width < 1560) {
```

Comment:
The logic on this conditional has changed. 
Given you've inverted it should now be 
```if (!vp || vp.width < 1560) { ```
... to be consistent with the previous logic.

@sndrs do you concur? For some reason I wasn't able to explain myself to @mxdvl last night so I might be barking up the wrong tree.

Reply:
Yes, I finally got it: we want to return `'f'` if we don't have a meaningful viewport object.